### DISCUSSION TITLE
2.0.6

### DISCUSSION THREAD

**Discussion by LuqueDaniel** [renpy/vscode-language-renpy#74] [Releases]
## What's Changed
* Allow language-renpy to run in an untrusted workspace. by @renpytom in https://github.com/LuqueDaniel/vscode-language-renpy/pull/70
* Removes folding provider in favor of indentation-based folding by @LuqueDaniel in https://github.com/LuqueDaniel/vscode-language-renpy/pull/72

## Fixes

* Fix folding class at the end of init python block [#68](https://github.com/LuqueDaniel/vscode-language-renpy/pull/68).
* Other issues related to folding: [#66](https://github.com/LuqueDaniel/vscode-language-renpy/issues/66) and [#65](https://github.com/LuqueDaniel/vscode-language-renpy/issues/65).

## New Contributors
* @renpytom made their first contribution in https://github.com/LuqueDaniel/vscode-language-renpy/pull/70

**Full Changelog**: https://github.com/LuqueDaniel/vscode-language-renpy/compare/v2.0.5...v2.0.6

<hr /><em>This discussion was created from the release <a href='https://github.com/LuqueDaniel/vscode-language-renpy/releases/tag/v2.0.6'>2.0.6</a>.</em>

**Comment by ghost** [comment]
Hi. Translated your file (readme - Ren'Py Language for Visual Studio Code) into Russian so that users can use your product.
